recipes/README.md

```diff
@@ -4,6 +4,18 @@
 1. [Management✅](management/): starting to come together with some useful content.
 
 # "Leadership" vs "Management"
+'Leadership' and 'management' are hard to define & distinguish. It's almost tempting to throw up our hands, give up, and shove all the recipes together.
+
+But there _is_ a difference, and we believe that not attempting to untangle the best leader patterns from the best manager patterns hurts our ability to grow the next generation of both. An illustration of this confusion comes from [TWIG](need_link_here), the Engineering Leadership class that Dobromir founded at Twitter. This was a course aimed at Individual Contributors, not managers, and yet a frequent question was "should I take this class if I want to become a manager?" One of the few strong, uncompromising positions we'll take in this cookbook is that _leadership is not just for people managers_. We strongly believe that teams cannot become [High Performing](need_link_here) without _every_ member's leadership voice. 
+
+So, as a very rough cut, recipes that mainly apply to people managers will go under the [management](management/) tree, while all other recipes will go under the [leadership](leadership/) tree. When there is ambiguity on where to find a set of recipes we will link them from both sides to make navigation easier.
+
+
+# Explaining the Difference
+
+We've seen the question "what is the difference between Leadership & Management?" come up often enough that it deserves its own set of recipes. Please contribute your own explanation here, or link to really good ones you find online!
+
+## Dobromir's Recipe
 
 Both Leadership & Management are fundamentally about changing how people think. Change how they think, and they'll
 behave differently. Change how they behave, and they'll reach different outcomes. Like the physics equivalents of
```

Comment:
this file has a different line break than the other ones - is this something you want to fix? (that is - you have lines broken mid-paragraph here, while the other docs keep single-line sentences... )

Reply:
I'm going to run scripts to clean the formatting up in the future so I'm not worrying about it yet.